www/_posts/2021-08-25-translations.md

```diff
@@ -0,0 +1,17 @@
+### Declaring Translation Bankruptcy
```

Comment:
I think the title should be added this way.

The build process will extract this information from this structure and fill in the title, author name, link, etc.

```suggestion
---
layout: post
author:
    name: Jesse MacFadyen
    url: https://twitter.com/purplecabbage
title:  "Declaring Translation Bankruptcy"
categories: announcements
tags: news
---

```

E.g. https://github.com/apache/cordova-docs/pull/1195/files#diff-5a1b10ca1420243c3bc1f16b19d52232f72ea7e44c803cf9b191e1bdb65f1b03R1-R8